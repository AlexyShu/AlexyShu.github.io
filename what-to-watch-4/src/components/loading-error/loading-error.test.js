import React from "react";
import renderer from "react-test-renderer";
import LoadingError from "./loading-error.jsx";

describe(`Render correct LoadingError`, () => {
  it(`Render LoadingError`, () => {
    const tree = renderer
    .create(<LoadingError />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
