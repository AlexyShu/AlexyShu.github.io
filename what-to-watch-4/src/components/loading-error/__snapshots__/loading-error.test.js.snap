// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correct LoadingError Render LoadingError 1`] = `
<div
  className="user-page"
>
  <h1
    className="page-title user-page__title"
  >
    Oh... NO! ðŸ˜³  Error! Please try again later.
  </h1>
</div>
`;
